src/components/CustomTable/StyledTableBody/StyledTableBody.js

```diff
@@ -27,12 +26,20 @@ const StyledTableBody = ({ columns, data, onRowClick }) => {
           classes={{ root: classes.root, hover: classes.hover }}
           onClick={() => onRowClick(row)}
         >
-          {columns.map((column) => (
-            <StyledTableCell
-              key={row.id + column.header}
-              value={get(row, column.field, "-")}
-            />
-          ))}
+          {columns.map((column) => {
+            const { header, field, cellRenderer } = column
+
+            if (!cellRenderer) return
```

Comment:
All react components must return a react node, this will crash the app if `cellRenderer` is not given, which is fine, but you should not be doing this.